gadgets/trace_dns/test/integration/trace_dns_test.go

```diff
@@ -156,6 +159,9 @@ func TestTraceDNS(t *testing.T) {
 					Timestamp: utils.NormalizedStr,
 					ID:        utils.NormalizedStr,
 					Latency:   0,
+					Truncated: false,
+					RecursionDesired: false,
```

Comment:
```suggestion
					RecursionDesired: true,
```
nslookup by default sets ra=true!

Reply:
Yes I sent that by mistake Sorry, Will make changes now. :)